gqueries/output_elements/output_series/user_uploaded_profiles/hydrogen_export_curve.gql

```diff
@@ -3,7 +3,7 @@
       V(
         FILTER(
         ALL(),
-        "hydrogen && (hydrogen.profile == :hydrogen_export)"
+        "hydrogen && (hydrogen.group == :hydrogen_export)"
```

Comment:
This does not match the curve on the `energy_export_hydrogen_baseload`, though in this case we should change the node itself. Good catch!